app/config.py

```diff
@@ -358,6 +359,21 @@
 # Amazon SES
 AWS_SES_REGION_NAME = os.environ.get("AWS_SES_REGION_NAME")
 
+# SMTP Auth Method (0:PASSWORD, 1:XOAUTH2)
+SMTP_AUTH_METHOD: int = int(os.environ.get("SMTP_AUTH_METHOD", 0))
```

Comment:
[imo] I thought it would be better to follow the RFC standard naming conventions and call them 0:PLAIN and 1:XOAUTH2.

Reply:
Thanks for the suggestion.

After checking the implementation, smtplib.login seems to automatically choose the appropriate mechanism (CRAM-MD5 / PLAIN / LOGIN) based on the server when a password is provided, so the current password handling may be acceptable.
Ref: https://github.com/python/cpython/blob/v3.13.11/Lib/smtplib.py#L734-L738